src/common/components/KeypomLogo/KeypomLogo.tsx

```diff
@@ -1,12 +1,15 @@
 import { HStack, Box, Text } from '@chakra-ui/react';
+import NextLink from 'next/link';
 
 export const KeypomLogo = () => {
   return (
-    <HStack spacing={{ base: 2, md: '2.5' }}>
-      <Box bgColor="gray.800" h={{ base: 5, md: '7' }} rounded="full" w={{ base: 5, md: '7' }} />
-      <Text as="b" fontSize={{ base: 'xl', md: '2xl' }}>
-        Keypom
-      </Text>
-    </HStack>
+    <NextLink href="/">
```

Comment:
<img width="427" alt="CleanShot 2022-12-21 at 18 33 49@2x" src="https://user-images.githubusercontent.com/6003064/208884541-9dbe91f3-1b4f-441c-b691-b7e0a48f4d56.png">

Currently the logo renders as `div` instead of link a link. Since this is a link to a page, should we make it a link so it doesn't affect SEO or accessibility? I'm not 100% sure how the new next/link will want us to use anchor tags with next link. Could you take a look?

Reply:
Actually Chakra has some info on nextjs 13 link. I'm not sure if we should be using chakra-ui's Link component but they do have a good example there 

https://chakra-ui.com/docs/components/link/usage#usage-with-nextjs

Reply:
If you intend to use chakra's link component here, do make the change to the rest of the PR as well!

Reply:
Oh I just realised this proj is still on Nextjs 12. If we're not upgrading yet, the same link to chakra's doc has the recommendation for how they do links pre-nextjs-13!

Reply:
Ya so I added `a` tag for SEO purposes, reason why I didn't use Chakra's Link is because Chakra Link underlines `Keypom` on hover, which is an unintended effect. Anyway, I added `as="a"` to the HStack component. This should address your concern!
<img width="834" alt="image" src="https://user-images.githubusercontent.com/19740800/208931766-469f0f28-3509-4b8b-9155-8a1195f2418c.png">